en/payloads/README.md

```diff
@@ -24,55 +24,65 @@ PX4 supports the following set of MAVLink commands/mission items for cameras tha
+:::note
+Control allocation is supported from PX4 v1.13 but disabled by default.
+:::
 
-The RC channels are *usually* mapped to the `AUX1`, `AUX2`, `AUX3` outputs of your flight controller (using a [mixer file](../concept/mixing.md) defined in your airfame).
+This is done in the [Actuators](../config/actuators.md#actuator-outputs) configuration screen by assigning the functions `Offboard Actuator Set 1` to `Offboard Actuator Set 6` functions to any desired payload outputs (in the [actuator outputs](../config/actuators.md#actuator-outputs) section).
+
+### Payload Outputs: Mixer Allocation
+
+The [mixer file](../concept/mixing.md) for the current airframe is used to specify the payload outputs for RC and MAVLink passthrough when control allocation is disabled.
+
+Usually just three outputs are mapped to the `AUX1`, `AUX2`, `AUX3` outputs of the flight controller.
 You can confirm which outputs are used for RC AUX passthrough on your vehicle in the [Airframe Reference](../airframes/airframe_reference.md).
-For example, [Quadrotor-X](../airframes/airframe_reference.md#quadrotor-x) has the normal mapping: "**AUX1:** feed-through of RC AUX1 channel", "**AUX2:** feed-through of RC AUX2 channel", "**AUX3:** feed-through of RC AUX3 channel".
+For example, [Quadrotor-X](../airframes/airframe_reference.md#quadrotor-x) has the normal mapping: 
+- "**AUX1:** feed-through of RC AUX1 channel"
+- "**AUX2:** feed-through of RC AUX2 channel"
+- "**AUX3:** feed-through of RC AUX3 channel"
 
 If your vehicle doesn't specify RC AUX feed-through outputs, then you can add them using using a custom [Mixer File](../concept/mixing.md) that maps [Control group 3](../concept/mixing.md#control-group-3-manual-passthrough) outputs 5-7 to your desired port(s).
 An example of such a mixer is the default passthrough mixer: [pass.aux.mix](https://github.com/PX4/PX4-Autopilot/blob/main/ROMFS/px4fmu_common/mixers/pass.aux.mix).
 
-:::note
-The same outputs used for "feed-through of RC AUX" may also be set using a MAVLink command (see [below](#mission-triggering)).
-PX4 will use the last value set through either mechanism.
-:::
 
+### RC Triggering
 
-### Mission Triggering
+The available passthrough (payload) outputs are mapped to RC channels using the [RC_MAP_AUXn](../advanced_config/parameter_reference.md#RC_MAP_AUX1) parameters.
 
-You can use the [MAV_CMD_DO_SET_ACTUATOR](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_SET_ACTUATOR) MAVLink command to set (up to) three actuators values at a time, either in a mission or as a command.
+If control allocation is used then `RC_MAP_AUXn` maps to the output that has an offboard acutator function with the same `n` index (e.g. `RC_MAP_AUX2` maps specified channel to the output with function `Offboard Actuator Set 2`).
```

Comment:
Same as above

Reply:
> Note that the ones for RC are the RC_AUX x functions

What do you mean?

What I've said is:
- You can map any output (AUX/MAIN/UAVCAN) to use for passthrough using actuator assignment of `Offboard Actuator Set x` if control allocation is enabled.
- You can similarly specify that some AUX outputs (only) are passthrough channels - e.g. the ones marked as "feed-through of RC AUX1 channel"
- You can map these to RC channels using - `RC_MAP_AUXn` where `n` and `x` (for the output) map 1:1.

I THINK perhaps you are saying that while you can map any output as passthrough, you can only use `RC_MAP_AUX` to map those outputs to RC channels if they are AUX outputs. There are no  `RC_MAP_MAIN` mappers, so you couldn't use RC passthrough for a non-AXU channel. Is that right?

I assume you can still use any outputs for mapping to the mavlink command to set an actuator?